### DISCUSSION TITLE
Filter for empty fields?

### DISCUSSION THREAD

**Discussion by Amtech281** [firecmsco/firecms#329] [Q&A]
Hi
We have in a collection a field which is sometimes filled and sometimes not, lets call it "MyField".
We want to filter as follows
a) Show all records where MyField is empty --> Could not find out how to do this
b) Show all records where MyField<>1.  --> It does not return documents having MyField empty.

Is it not possible to apply these filters?

thanks!

**Comment by fgatti675** [comment]
Hi @Andre3424 
So when you see an empty value in the CMS it may mean that the value is not defined (the field is not present at all in the database) or that is null. Unfortunately Firestore does not allow to list documents with the first case.

**Reply by Amtech281** [reply]
@fgatti675 thanks for your answer. So Firestore is also not able to retrieve documents where e.g. field is missing?

**Reply by fgatti675** [reply]
That is correct. But what you can do is running a script in your data to set your fields to `null`
